model-optimizer/extensions/ops/layer_norm.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from mo.front.common.partial_infer.elemental import copy_shape_infer
+from mo.graph.graph import Graph
+from mo.ops.op import Op
+
+
+class LayerNorm(Op):
+    """
+    LayerNorm - normalize along axis 1.
```

Comment:
According to MxNet specification https://mxnet.apache.org/versions/1.6/api/r/docs/api/mx.symbol.LayerNorm.html, `axis` can be any integer value, not only `1`.